pkg/kubeadm/util/tokens.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package kubeadmutil
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	kubeadmapi "k8s.io/kubernetes/pkg/kubeadm/api"
+	utilrand "k8s.io/kubernetes/pkg/util/rand"
+)
+
+const (
+	TokenIDLen = 6
+	TokenBytes = 8
+)
+
+func GenerateToken(params *kubeadmapi.BootstrapParams) error {
+	fmt.Println("Generating token...")
+	tokenID, token := utilrand.HexString(TokenIDLen), utilrand.HexString(TokenBytes*2)
```

Comment:
`k8s.io/kubernetes/pkg/util/rand` uses random number generators from `math/rand`.  That package says "For random numbers suitable for security-sensitive work, see the crypto/rand package."  We should do that.